src/cascadia/TerminalSettingsEditor/AddProfile.cpp

```diff
@@ -39,4 +39,13 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             _State.RequestDuplicate(selected.try_as<Model::Profile>().Guid());
         }
     }
+
+    void AddProfile::ProfilesSelectionChanged(const IInspectable& /*sender*/,
+                                              const Windows::UI::Xaml::RoutedEventArgs& /*eventArgs*/)
+    {
+        if (!IsProfileSelected())

```

Comment:
```suggestion
        if (!_IsProfileSelected)
```
nit: I think this'll be faster since we're not calling the function, technically